aea/aea_builder.py

```diff
@@ -292,6 +292,8 @@ class AEABuilder:
     DEFAULT_RUNTIME_MODE = "threaded"
     DEFAULT_SEARCH_SERVICE_ADDRESS = "fetchai/soef:*"
 
+    loader = ConfigLoader.from_configuration_type(PackageType.AGENT)
```

Comment:
OK for now, but in the long term we shouldn't need the instantiation of a loader, just a "pure function" that keeps the jsonschema loaded in memory (maybe lazily loaded)